chassis/core/src/main/java/com/griddynamics/jagger/util/generators/WorkloadGenerator.java

```diff
@@ -19,6 +27,20 @@ static WorkloadClockConfiguration generateLoad(JLoadProfile jLoadProfile) {
             ((RpsClockConfiguration) clockConfiguration).setValue(((JLoadProfileRps) jLoadProfile).getRequestsPerSecond());
             ((RpsClockConfiguration) clockConfiguration).setWarmUpTime(((JLoadProfileRps) jLoadProfile).getWarmUpTimeInSeconds());
             ((RpsClockConfiguration) clockConfiguration).setMaxThreadNumber((int) ((JLoadProfileRps) jLoadProfile).getMaxLoadThreads());
+        } else if (jLoadProfile instanceof JLoadProfileUserGroups) {
+            clockConfiguration = new UserGroupsClockConfiguration();
+            List<User> users = ((JLoadProfileUserGroups) jLoadProfile).getUserGroups().stream()
+                    .map(jLoadUser -> new User(String.valueOf(jLoadUser.getNumberOfUsers()), String.valueOf(jLoadUser.getSlewRateUsersPerSecond()),
+                            jLoadUser.getStartDelayInSeconds() + "s", "1s", jLoadUser.getLifeTimeInSeconds() + "s"))
+                    .collect(toList());
+
+            ((UserGroupsClockConfiguration) clockConfiguration).setUsers(users);
```

Comment:
Because clockConfiguration has type WorkloadClockConfiguration and must be casted to it's real type to set values